### ISSUE TITLE
Admonitions in preview

### ISSUE THREAD

**Issue by pihentagy** [silverbulletmd/silverbullet#868]
Admonitions are not displayed in Markdown preview

(maybe time to check all [Extensions](https://silverbullet.md/Markdown/Extensions))

**Comment by zefhemel** [comment]
Yep indeed, this was never implemented. Would be good to have 👍🏻

**Comment by pihentagy** [comment]
Is it difficult to implement this?

**Comment by MrMugame** [comment]
No [this](https://github.com/silverbulletmd/silverbullet/blob/main/web/cm_plugins/admonition.ts) just has to be implemented [here], but it's less complex because you don't have all the cursor stuff

**Comment by MrMugame** [comment]
`HTMLTag`, also doesn't get rendered

**Comment by MrMugame** [comment]
And also kinda related, this `> ![note**` parses as an admonition, which it really shouldn't I think. And maybe admonitions should be parsed as separate markdown nodes?
